schema/crdb/token-and-session-ids/up02.sql

```diff
@@ -0,0 +1,5 @@
+set local disallow_full_table_scans = off;
+
+UPDATE omicron.public.console_session
+  SET id = gen_random_uuid()
+  WHERE id IS NULL;
```

Comment:
is there a reason we can't do this by adding the column with:

```sql
ALTER TABLE omicron.public.console_session
  ADD COLUMN IF NOT EXISTS id UUID NOT NULL
  DEFAULT gen_random_uuid();
```
and then
```sql
ALTER TABLE omicron.public.console_session
   ALTER COLUMN id
   DROP DEFAULT;
```

that requires two transactions rather than three, and feels a little cleaner to me. if there's a reason we can't do it like that, it could be nice to know why.

not a big deal though, it should be fine either way.